package.json

```diff
@@ -115,8 +114,8 @@
     "typescript": "^4.3.2"
   },
   "peerDependencies": {
-    "ember-firebase-service": "^9.0.0",
-    "ember-simple-auth": "^3.0.0"
+    "ember-simple-auth": "^3.0.0",
```

Comment:
FYI, ESA just released 4.0.0 https://github.com/simplabs/ember-simple-auth/releases/tag/4.0.0

Reply:
Something I will update in a different PR. Thanks for the notice!